projects/cm_mcu/Tasks.h

```diff
@@ -170,6 +170,8 @@ struct dev_moni2c_addr_t {
   uint8_t mux_bit;              // port of the mux; write value 0x1U<<mux_bit to the mux register
   uint8_t dev_addr;             // I2C address of device.
   uint16_t eeprom_progname_reg; // register on eeprom for reading program version
+  const char *configname_chip;
+  const char *configname_eeprom;
```

Comment:
these three variables are not generic for all devices, are they? they are just for clock devices. Should they be stored elsewhere?

Reply:
You're right. I think having this information associated with a device struct is quite handy in a few places `projects/cm_mcu/commands/SensorControl.c`, `projects/cm_mcu/InitTask.c`, and `projects/cm_mcu/clocksynth.c` which need to access it and update/read. Unless you have other suggestions?

Reply:
Maybe we should create a way to store them

Reply:
They are varying by clock chips so I thought it made sense to have them be an attribute in a struct. Though, it's not clean for other non-clocks. Do you have an alternative way to store them?

Reply:
also there is no storage associated with this and no variables in global scope that this points to. Instead we point to some variables on the heap in `InitTask`. Maybe it makes sense to instead create a separate struct somewhere, with storage associated to it, and then put them in there.
If `InitTask` ever goes out of scope the pointers you set there will point to random addresses and contain possibly garbage.

Reply:
Done.